app/views/settings/profiles/show.html.haml

```diff
@@ -18,7 +21,13 @@
         = f.input :display_name, wrapper: :with_block_label, input_html: { maxlength: 30, data: { default: @account.username } }
 
       .fields-group
-        = f.input :note, wrapper: :with_block_label, input_html: { maxlength: 500 }
+        - account_note_maxlength = 500
```

Comment:
There is an `Account::NOTE_LENGTH_LIMIT` constant now available that this could from.